website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-snapshotting-source-freshness.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Source freshness"
 id: "cloud-snapshotting-source-freshness"
-description: "You can see the state of source data by loking at a recent snapshot."
+description: "Look at a recent snapshot to see the state of source data."
```

Comment:
"Snapshotting" terminology is actually deprecated (we have another feature called snapshots and this was confusing - see [the info box on this page](https://docs.getdbt.com/reference/commands/source)). This might need a wider re-work?
```suggestion
description: "Validate that data freshness meets expectations and alert if stale."
```